Flaky-test: DelayedDeliveryTest.testEnableTopicDelayedDelivery

### Search before asking

- [x] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.


### Example failure

https://github.com/apache/pulsar/actions/runs/12923783677/job/36041928951?pr=23874#step:11:1423

### Exception stacktrace

<!-- copy-paste the stack trace in the code block below -->
```
  Error:  org.apache.pulsar.broker.service.persistent.DelayedDeliveryTest.testEnableTopicDelayedDelivery  Time elapsed: 6.941 s  <<< FAILURE!
  java.lang.AssertionError: expected [null] but found [org.apache.pulsar.client.impl.TopicMessageImpl@728268b3]
  	at org.testng.Assert.fail(Assert.java:110)
  	at org.testng.Assert.failNotSame(Assert.java:1573)
  	at org.testng.Assert.assertNull(Assert.java:1506)
  	at org.testng.Assert.assertNull(Assert.java:1494)
  	at org.apache.pulsar.broker.service.persistent.DelayedDeliveryTest.testEnableTopicDelayedDelivery(DelayedDeliveryTest.java:488)
  	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
  	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
  	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
  	at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
  	at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
  	at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
  	at java.base/java.lang.Thread.run(Thread.java:1583)
```


### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

--- a/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/DelayedDeliveryTest.java
+++ b/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/DelayedDeliveryTest.java
@@ -483,10 +483,10 @@ public class DelayedDeliveryTest extends ProducerConsumerBase {
                 break;
             }
         }
-        producer.newMessage().value("long-tick-msg").deliverAfter(2, TimeUnit.SECONDS).send();
+        producer.newMessage().value("long-tick-msg").deliverAfter(3, TimeUnit.SECONDS).send();
         msg = consumer.receive(1, TimeUnit.SECONDS);
         assertNull(msg);
-        msg = consumer.receive(3, TimeUnit.SECONDS);
+        msg = consumer.receive(4, TimeUnit.SECONDS);
         assertNotNull(msg);
     }
 

